OxfordBodleian/Juel-Jensen/BDLaethe35.xml

```diff
@@ -227,6 +751,157 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                            ዘንዕዶ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                            ሱራሄ፡ መለኮት፡ ዘየዓውዶ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                            <hi rend="rubric">ሰንበተ፡</hi> ክር፡ ቅድስት፡ እስእለኪ፡ 
+                                            በአፈድፍዶ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                            ለዘይስእለኪ፡ ሥላሴ፡ ክሂዶ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                            መብረቀ፡ ሰማይ፡ ይህርይ፡ እምሰማይ፡ ወሪዶ<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                            <hi rend="rubric">ስብሐት፡</hi> ለኪ፡ ሰንበታዊ፡ ለነፍሰ፡ ኃጥአን፡ ዕረፍት፡
+                                            <hi rend="rubric">ስብሐ</hi><supplied reason="omitted">፡</supplied> ለኪ፡ ነዓዌ፡ ኵሉ፡ አራዊት።
+                                            <gap reason="ellipsis" unit="chars" quantity="2"/> ለኪ፡ 
+                                            ሰንበታዊት<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                            <hi rend="rubric">ስብሐ</hi><supplied reason="omitted">፡</supplied> ለኪ፡ በኍልቈ፡ ዕለታት፡ 
+                                            ወአዝማናት<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                            <hi rend="rubric">ስብሐ</hi><supplied reason="omitted">፡</supplied> ይደሉ፡ ለመንግሥትኪ፡ 
+                                            ቡርክት<hi rend="rubric" rendition="#partialRubric">፨</hi>
+                                            ወለወልድኪ፡ ይደሉ፡ ስብሐት<hi rend="rubric" rendition="#partialRubric">፨ ፨ ፨ ፨</hi>
+                                        </seg>
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a4">
+                                    <locus target="#4r"/>
+                                    <desc type="Unclear">Introduction to <ref target="#ms_i1.1"/> added secondarily in red.</desc>
+                                    <q xml:lang="gez">
+                                        <seg part="I">
+                                            <hi rend="rubric">ነዓ፡ ኀቤ<gap reason="illegible" unit="chars" quantity="1"/>፡ ዳዊት፡ ንጉሠ፡ እስራኤል፡</hi>
+                                        </seg>
+                                        <seg part="F">
+                                            <hi rend="rubric">እንዘ፡ እፀርሕ፡ ወእብል፡</hi>
+                                        </seg>
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a5">
+                                    <desc type="findingAid">Scribbles in pen, e.g. on <locus target="#4r #5r #6r #53r"/>;
+                                        Gǝʿǝz names of the week in some of the latter parts of the manuscript:
```

Comment:
Sometimes Gǝʿǝz  and sometimes Geez is used. I think it would be good to standardize now to Gǝʿǝz for easier searchability, and then we can discuss our preferred spelling in the next meeting